lib/spack/docs/external_rocm.rst

```diff
@@ -0,0 +1,84 @@
+.. Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+   Spack Project Developers. See the top-level COPYRIGHT file for details.
+
+   SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+===================================
+Using an External ROCm Installation
```

Comment:
Can you make this a subsection of a top-level "GPU Configuration" section?  It can just remain AMD-specific for now but the idea would be to also have example configuration for oneapi and cuda.

Reply:
@tgamblin I generalized it (and added a short section on CUDA)